File: httpsnippet/src/test/resources/output/clojure/clj_http/multipart-form-data.clj
(require '[clj-http.client :as client])

(client/post "http://mockbin.com/har" {headers {content-type "multipart/form-data" 
                                         } 
                                      :multipart [{name "foo" 
                                           :content "bar" 
                                           } ] 
                                      })

File: httpsnippet/src/test/resources/output/python/python3/multipart-form-data.py
import http.client

conn = http.client.HTTPConnection("mockbin.com", "80")

payload = {"foo":"bar"}

headers = {
	"content-type": "multipart/form-data"
}

conn.request("POST", "/har", payload, headers)

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))